downloads.md

```diff
@@ -27,19 +53,39 @@ window.onload = function () {
 3. Download Spark: <span id="spanDownloadLink"></span>
 
 4. Verify this release using the <span id="sparkDownloadVerify"></span> and [project release KEYS](https://downloads.apache.org/spark/KEYS) by following these [procedures](https://www.apache.org/info/verification.html).
+classic
 
-Note that Spark 3 is pre-built with Scala 2.12 in general and Spark 3.2+ provides additional pre-built distribution with Scala 2.13.
+Note that Spark 4 is pre-built with Scala 2.13 in general, and Spark 3.5+ provides additional pre-built distribution with Scala 2.13.
 
-### Link with Spark
+### Link with Spark ###
 Spark artifacts are [hosted in Maven Central](https://search.maven.org/search?q=g:org.apache.spark). You can add a Maven dependency with the following coordinates:
 
     groupId: org.apache.spark
-    artifactId: spark-core_2.12
-    version: 3.5.4
+    artifactId: spark-core_2.13
+    version: 4.0.0
+
+### Installing with PyPi ###
+Like the two distributions mentioned above, PyPI will also have two PySpark package versions. The default is the _classic_ __pyspark__, while the _connect_ version is __pyspark-connect__ and is dependent on __pyspark__.
+
+Use the decision matrix above to select which PyPI PySpark package to use for your Spark workloads. Both <a href="https://pypi.org/project/pyspark/">PySpark</a> package versions are available on PyPI.
+
+### Installing PySpark Connect ###
+
+Since __pyspark-connect__ package is dependent on __pyspark__, __pyspark-connect__ will automatically install __pyspark__ for you. The __pyspark-connect__ package is mostly empty and merely enables Spark config `spark.api.mode` to _connect_ mode in the underlying pyspark package.
+
+`pip install pyspark-connect==4.0.0`
+
+Thereafter, follow the Spark Connect [quickstart guide](https://spark.apache.org/docs/4.0.0-preview2/api/python/getting_started/quickstart_connect.html) on how to use SparkSession.
+
+### Installing PySpark Classic ###
+
+Simply use `pip install pyspark==4.0.0`
```

Comment:
I would suggest reordering these - until adoption increases.
Keep Classic above Connect

Reply:
@mridulm Thanks for the feedback. The order here is not intentionally to lead or steer one to use _connect_ instead of _classic_. Instead, it is easier and simpler to be more expansive and comprehensive, switching between modes as the last. 

It's much easier to read and follow the decision matrix than the reverse.

Reply:
I would suggest reordering it.